### PR TITLE
Don't setup Python doc packages every time

### PR DESCRIPTION
**Pull request by soltanmm-google**
It's silly.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -207,10 +212,14 @@ def cython_extensions_and_necessity():
   INSTALL_REQUIRES += ('futures>=2.2.0',)
 
 SETUP_REQUIRES = INSTALL_REQUIRES + (
+    'six>=1.10',
+) + (
+  (
     'sphinx>=1.3',
     'sphinx_rtd_theme>=0.1.8',
-    'six>=1.10',
```
Comment (kpayson64) [inline setup.py]: I think six>=1.10.0 is imposed by sphinx, so we can probably leave this here.

Comment (soltanmm-google) [inline setup.py]: aye

**Comment by nathanielmanistaatgoogle** [comment]
In your commit log message: "setup" is a noun; "set up" is a verb.

**Comment by soltanmm-google** [comment]
fix'd

**Pull request merged**